admiral/pkg/apis/admiral/model/globalrouting.proto

```diff
@@ -74,6 +74,19 @@ message TrafficPolicy {
     //REQUIRED: dnsPrefix that will be prefixed for the service names being generated with this traffic policy
     //Ex: dnsPrefix = west => generated service name = west.stage.servicename.global
     string dnsPrefix = 4;
+
+   message FailoverConfiguration {
+       //REQUIRED: Minimum duration of time in seconds, the endpoint will be ejected
+       int64 base_ejection_time = 1;
+       //REQUIRED: No. of consecutive failures in specified interval after which the endpoint will be ejected
+       uint32 consecutive_gateway_errors = 2;
+       //REQUIRED: Time interval between ejection sweep analysis
+       int64 interval = 3;
+   }
+
+   //OPTIONAL: to configure the outlierDetection in DestinationRule
+   FailoverConfiguration failover_configuration = 5;
```

Comment:
Should we call this `outlierDetection` ? Just like in https://istio.io/latest/docs/reference/config/networking/destination-rule/#OutlierDetection

Reply:
If we do plan to change it to outlierDetection, then will be good to update references of failure configuration or failure_configuration in test cases.